### PR TITLE
[Fix] Attempt to read property name on bool

### PR DESCRIPTION
**Pull request by mebishalnapit**
This PR includes:

* PHP warning to be logged into debug log file if **(AnsPress) Breadcrumbs** widget is used in the **(AnsPress) Question List** Top widgetized area and viewing the individual tag page
* Fixes the display of the single tag item in the single tag page and Tags item in the tags archive page for the **(AnsPress) Breadcrumbs** widget
* Fixes the condition to check for currently viewing page is either a tag page or tags archive page

### CODE REVIEW

File: addons/tags/tags.php [hunk 1]
```diff
@@ -443,7 +443,7 @@ public function page_title( $title ) {
 			$title = ap_opt( 'tags_page_title' );
 		} elseif ( is_question_tag() ) {
 			$tag_id = sanitize_title( get_query_var( 'q_tag' ) );
-			$tag = get_term_by( 'slug', $tag_id, 'question_tag' ); // @codingStandardsIgnoreLine.
+			$tag    = $tag_id ? get_term_by( 'slug', $tag_id, 'question_tag' ) : get_queried_object();
 			$title  = $tag->name;
```
Comment (rahularyan) [inline addons/tags/tags.php:447]: @mebishalnapit Add a if check here.

File: addons/tags/tags.php [hunk 2]
```diff
@@ -459,7 +459,7 @@ public function page_title( $title ) {
 	public function ap_breadcrumbs( $navs ) {
 		if ( is_question_tag() ) {
 			$tag_id       = sanitize_title( get_query_var( 'q_tag' ) );
-			$tag = get_term_by( 'slug', $tag_id, 'question_tag' ); // @codingStandardsIgnoreLine.
+			$tag          = $tag_id ? get_term_by( 'slug', $tag_id, 'question_tag' ) : get_queried_object();
```
Comment (rahularyan) [inline addons/tags/tags.php:462]: @mebishalnapit Add if check after this to make sure tag is an object or term.

**Review by rahularyan — CHANGES_REQUESTED** [review]

File: addons/tags/tags.php (hunk 1 above)
Comment (mebishalnapit) [reply addons/tags/tags.php]: @rahularyan Added.

File: addons/tags/tags.php (hunk 2 above)
Comment (mebishalnapit) [reply addons/tags/tags.php:465]: @rahularyan It should already be available here: https://github.com/anspress/anspress/blob/bc169e9242ce91dfeceb0642db304408352a2e88/addons/tags/tags.php#L469-L475

**Review by rahularyan — APPROVED** [review]

**Pull request merged**